### PR TITLE
[stable/datadog] Allow configuration of daemonset.updateStrategy for datadog chart

### PR DESCRIPTION
**Pull request by tvanhens**
* Allows the updateStrategy to be set with a default of "OnDelete" set for backwards compatibility

### CODE REVIEW

File: stable/datadog/templates/daemonset.yaml [hunk 1]
```diff
@@ -97,5 +97,7 @@ spec:
       tolerations:
 {{ toYaml .Values.daemonset.tolerations | indent 8 }}
       {{- end }}
+  updateStrategy:
+    type: {{ default "OnDelete" .Values.daemonset.updateStrategy | quote }}
```
Comment (dhilipkumars) [inline stable/datadog/templates/daemonset.yaml:101]: i cant find `updateStrategy ` in the `.Values` files, Im i missing something?

**Review by dhilipkumars — CHANGES_REQUESTED** [review]

File: stable/datadog/templates/daemonset.yaml (hunk 1 above)
Comment (tvanhens) [reply stable/datadog/templates/daemonset.yaml:101]: Nope not missing anything. I am though. PR has been updated with a commented out line to document rolling updates.

**Review by dhilipkumars — APPROVED** [review]

**Comment by dhilipkumars** [comment]
@tvanhens Thanks for the contribution. :-)

**Pull request merged**